File: WIS/source/WIS_Prototype/WIS_Prototype/src/app/report/transfer-report.ts
export class TransferReport {
  weightsheetId: bigint;
  sourceName: string;
  commodityTypeName: string;
  commodityVarietyName: string;
  netWeight: number;
  numLoads: number;

  constructor() {
    this.weightsheetId =BigInt(0);
    this.sourceName = '';
    this.commodityTypeName = '';
    this.commodityVarietyName = '';
    this.netWeight = 0;
    this.numLoads = 0;
  }
}

File: WIS/source/WIS_Prototype/WIS_PrototypeAPI/Data/DTOs/TransferReport.cs
namespace WIS_PrototypeAPI.Data.DTOs
{
	public class TransferReport
	{
		public long WeightsheetId { get; set; }
		public string SourceName { get; set; }
		public string CommodityTypeName { get; set; }
		public string CommodityVarietyName { get; set; }
		public int NetWeight { get; set; }
		public int NumLoads { get; set; }
	}
}
